src/features/rewards/thumbLikePicker/style.ts

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License. v. 2.0. If a copy of the MPL was not distributed with this file.
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const StyledAdStatThumbUpIcon = styled<{}, 'div'>('div')`
+  display: inline-block;
+  width: 32px;
+  height: 32px;
+  padding: 4px;
+  cursor: pointer;
+`
+
+export const StyledAdStatThumbUpFilledIcon = styled(StyledAdStatThumbUpIcon)`
+  color: red;
```

Comment:
please use theme color

Reply:
Fixed.

Reply:
not seeing the change, could you please check again